athena-slack-member-analytics/src/main/java/com/amazonaws/connectors/athena/slack/util/SlackSchemaUtility.java

```diff
@@ -0,0 +1,244 @@
+/*-
+ * #%L
+ * athena-example
+ * %%
+ * Copyright (C) 2019 - 2020 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.connectors.athena.slack.util;
+
+import com.amazonaws.athena.connector.lambda.records.ReadRecordsRequest;
+import com.amazonaws.athena.connector.lambda.data.writers.extractors.IntExtractor;
+import com.amazonaws.athena.connector.lambda.data.writers.extractors.VarCharExtractor;
+import com.amazonaws.athena.connector.lambda.data.writers.holders.NullableVarCharHolder;
+import com.amazonaws.athena.connector.lambda.data.SchemaBuilder;
+import com.amazonaws.services.secretsmanager.AWSSecretsManager;
+import com.amazonaws.services.secretsmanager.AWSSecretsManagerClientBuilder;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueRequest;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueResult;
+import org.apache.arrow.vector.holders.NullableIntHolder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.json.JSONObject;
+import org.json.JSONArray;
+import java.util.Base64;
+import com.amazonaws.athena.connector.lambda.data.writers.GeneratedRowWriter;
+
+public class SlackSchemaUtility {
+    
+    private static final Logger logger          = LoggerFactory.getLogger(SlackSchemaUtility.class);
+    private static final String TYPE_INTEGER    = "int";
+    private static final String TYPE_STRING     = "string";
+    private static final String TYPE_FLOAT4     = "float";
+    private static final String TYPE_STRUCT     = "struct";
+    private static final String TYPE_LIST       = "list";
+    
+    /**
+     * Retrieves json schema for a particular table.
+     * 
+     * @param tableName  String with the name of the table.
+     * @return JSONObject with the table schema
+     * 
+     */
+    public static JSONObject getSchema(String tableName){
+        
+        logger.info("getSchema: enter - " + tableName);
+        
+        // TODO - Get sample data from endpoint. For the slack member analytics endpoint
```

Comment:
Lets address this TODO before considering for merge

Reply:
Implemented change